<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionClientDialog</class>
 <widget class="QDialog" name="GestionClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Ajouter / modifier un client</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_details_client">
     <property name="font">
      <font>
       <family>Outfit</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {border: none; border-radius: 4px;}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_nom_client">
        <property name="font">
         <font>
          <family>Outfit</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nom:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_nom_client">
        <property name="font">
         <font>
          <family>Outfit</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
border: 1px solid rgb(180, 180, 180);
border-radius: 4px;
padding: 2px 5px;
}

QLineEdit:focus {
border: 1px solid rgb(85, 170, 255);
}</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_numero_client">
        <property name="font">
         <font>
          <family>Outfit</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Téléphone:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="input_telephone_client">
        <property name="font">
         <font>
          <family>Outfit</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
border: 1px solid rgb(180, 180, 180);
border-radius: 4px;
padding: 2px 5px;
}

QLineEdit:focus {
border: 1px solid rgb(85, 170, 255);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>33</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_annuler">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">

QPushButton {
background-color:  rgba(217, 217, 217, 100);
 padding: 5px 10px;
 border: 2px solid rgb(102, 102, 102);
border-radius: 4px;
color:   rgb(102, 102, 102); 
font: bold;
}

QPushButton:hover {
	background-color:    rgba(217, 217, 217, 150) ; 
}</string>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_enregistrer">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
background-color:  rgba(87, 182, 255, 50);
 padding: 5px 10px;
 border: 2px solid rgb(0, 85, 255);
border-radius: 4px;
color: rgb(0, 85, 255); 
font: bold;
}

QPushButton:hover {
	background-color:  rgba(87, 182, 255, 80) ; 
}</string>
       </property>
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
